corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditorMain.java

```diff
@@ -182,17 +184,25 @@ public static void main(String[] args) {
                     browser.editRecord(namespace, tableName, keyToEdit, newRecord);
                     break;
                 case deleteRecord:
-                    String keyToDelete = null;
-                    if (opts.get("--keyToDelete") != null) {
-                        keyToDelete = String.valueOf(opts.get("--keyToDelete"));
-                    }
+                    final String keyToDeleteStr = "--keyToDelete";
+                    final String keysToDeleteStr = "--keysToDelete";
                     Preconditions.checkArgument(isValid(namespace),
                             "Namespace is null or empty.");
                     Preconditions.checkArgument(isValid(tableName),
                             "Table name is null or empty.");
-                    Preconditions.checkNotNull(keyToDelete,
-                            "Key To Delete is Null.");
-                    browser.deleteRecord(namespace, tableName, keyToDelete);
+                    String keyToDelete = null;
+                    if (opts.get(keyToDeleteStr) != null) {
+                        keyToDelete = String.valueOf(opts.get(keyToDeleteStr));
+                        Preconditions.checkNotNull(keyToDelete,
+                                "Key To Delete is Null.");
+                        browser.deleteRecords(namespace, tableName, Arrays.asList(keyToDelete));
```

Comment:
Why not just have keysToDelete?
keyToDelete is anyhow using deleteRecords and is passed as a List. keyToDelete would also be a JSON ultimately.

Reply:
`keysToDelete` is a file path while `keyToDelete` is a json string right. Maybe I will change the name so it isn't confusing.